src/box/lua/init.c

```diff
@@ -169,15 +169,15 @@ extern char session_lua[],
 	config_utils_odict_lua[],
 	config_utils_schema_lua[],
 	config_utils_snapshot_lua[],
-	config_utils_tabulate_lua[]
+	config_utils_tabulate_lua[],
 #if ENABLE_CONFIG_EXTRAS
-	,
 	config_source_etcd_lua[],
 	config_storage_init_lua[],
 	config_source_storage_lua[],
-	config_extras_lua[]
+	config_extras_lua[],
 #endif
-	;
+	experimental_connpool_lua[];
+
 	/* }}} config */
```

Comment:
Nit: This new item should be outside of the `/* {{{ config */` `/* }}} config */` block.

Reply:
Fixed.